### PR TITLE
Dynamic advanced links list

### PR DESCRIPTION
**Pull request by brichet**
This PR allows to retrieves the available advanced links dynamically, using a rest API.

- adds a rest API handler 
- removes the default categories from `AdvancedLinkingChoices` in favor to the one get from API

### CODE REVIEW

**Comment by martinRenou** [comment]
@brichet is it ready for review?

**Comment by brichet** [comment]
> @brichet is it ready for review?

Yes

**Comment by martinRenou** [comment]
Check link is binder being down, we can ignore

File: glue_lab/handlers.py [hunk 1]
```diff
@@ -4,20 +4,27 @@
...
 class RouteHandler(APIHandler):
     # The following decorator should be present on all verb methods (head, get, post,
     # patch, put, delete, options) to ensure only authorized user can request the
     # Jupyter server
     @tornado.web.authenticated
-    def get(self):
-        self.finish(json.dumps({"data": "This is /glue-lab/get_example endpoint!"}))
+    def get(self, path):
+        if path == "available-advanced-links":
+            self.finish(json.dumps({"data": get_advanced_links()}))
```
Comment (trungleduc) [inline glue_lab/handlers.py:17]: I think we can cache the result of `get_advanced_links` instead of calling it on every request.

**Comment by brichet** [comment]
Thanks @martinRenou for rebasing it

**Pull request merged**